modules/33acrossBidAdapter.js

```diff
@@ -526,14 +578,19 @@ function _isIframe() {
 }
 
 // **************************** INTERPRET RESPONSE ******************************** //
-// NOTE: At this point, the response from 33exchange will only ever contain one bid
-// i.e. the highest bid
 function interpretResponse(serverResponse, bidRequest) {
   const bidResponses = [];
-
-  // If there are bids, look at the first bid of the first seatbid (see NOTE above for assumption about ttx)
-  if (serverResponse.body.seatbid.length > 0 && serverResponse.body.seatbid[0].bid.length > 0) {
-    bidResponses.push(_createBidResponse(serverResponse.body));
+  const { seatbid, cur } = serverResponse.body;
+
+  if (seatbid.length > 0) {
```

Comment:
do not need this check here, the forEach() will not run if the array is empty.

same for line 587